TFT/src/User/API/config.c

```diff
@@ -951,21 +945,16 @@ void parseConfigKey(uint16_t index)
       #endif
 
       case C_INDEX_NEOPIXEL_PIXELS:
-      {
-        uint16_t pixels = config_int();
-        if (inLimit(pixels, 0, MAX_NEOPIXEL_PIXELS))
-        {
-          infoSettings.neopixel_pixels = pixels ? pixels : NEOPIXEL_PIXELS;
-        }
+        SET_VALID_INT_VALUE(infoSettings.knob_led_idle, 0, MAX_NEOPIXEL_PIXELS);
```

Comment:
should be SET_VALID_INT_VALUE(`infoSettings.neopixel_pixels`, 0, MAX_NEOPIXEL_PIXELS); not SET_VALID_INT_VALUE(`infoSettings.knob_led_idle`, 0, MAX_NEOPIXEL_PIXELS); ?

Reply:
Yes it should be. Bad copy paste.